container/container.go

```diff
@@ -914,6 +916,19 @@ func (container *Container) BuildCreateEndpointOptions(n libnetwork.Network, epC
 		}
 	}
 
+	var dns []string
+
+	if len(container.HostConfig.DNS) > 0 {
+		dns = container.HostConfig.DNS
+	} else if len(epConfig.DNS) > 0 {
+		dns = epConfig.DNS
```

Comment:
@mavenugo  if I remove epConfig.DNS then we can't support daemon.configStore.DNS without a change in the public API or overwriting container.HostConfig.DNS. What option do you think I should take? Should we drop the support for daemon.configStore.DNS altogether?